src/common/SPELLS/DEATH_KNIGHT.js

```diff
@@ -410,12 +410,18 @@ export default {
   },
   
   // Unholy Tier Sets
-  // T20 2 Piece
-  UNHOLY_DEATH_KNIGHT_T20_2SET_BONUS_BUFF: {
-    id: 246995,
-    name: 'T20 2 Set Bonus',
+  // T20 2 Piece and buff
+  UNHOLY_DEATH_KNIGHT_T20_2SET_BONUS: {
```

Comment:
``and buff``` ? Surely it's only the t202p

Reply:
The intent is that its describing the next 2 entries in the spell file.  When I add T21 there'll be a new comment after master of ghouls indicating that the T21 spells are beginning.  The way the file is laid out, each comment defines everything under it until theres a new comment